examples/resources/env0_template_project_assignment/resource.tf

```diff
@@ -0,0 +1,20 @@
+data "env0_project" "default_project" {
+  name = "Default Organization Project"
+}
+
+data "env0_ssh_key" "my_key" {
```

Comment:
please remove the ssh key from here it's not part of this thing

Reply:
747cfb10

Reply:
939db789